package/MDAnalysis/analysis/msd.py

```diff
@@ -290,6 +295,8 @@ class EinsteinMSD(AnalysisBase):
         The averaged MSD over all the particles with respect to lag-time.
     results.msds_by_particle : :class:`numpy.ndarray`
         The MSD of each individual particle with respect to lag-time.
+    results.delta_t_values : :class:`numpy.ndarray`
+        Array of unique Δt (time differences) at which time-averaged MSD values are computed.
```

Comment:
Indicate when this new results attribute was added so that users understand which version of the code they need
```suggestion
        Array of unique Δt (time differences) at which time-averaged MSD values are
        computed.

        .. versionadded: 2.10.0

```